src/main/java/teammates/ui/webapi/UpdateFeedbackSessionAction.java

```diff
@@ -42,12 +44,33 @@ void checkSpecificAccessControl() throws UnauthorizedAccessException {
     public JsonResult execute() throws InvalidHttpRequestBodyException {
         String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
         String feedbackSessionName = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        // TODO: Use the value here.
+        getBooleanRequestParamValue(Const.ParamsNames.NOTIFY_ABOUT_DEADLINES);
```

Comment:
I assume this will be used by dependent PR?

Reply:
Yes, this will be passed to the `DeadlineExtensionWorker` of my PR. We are planning on integrating them in a subsequent PR.